shell/ci/release/trigger-rc-release.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+# This script is to add chore: Release comit to the default pre-release
+# branch to trigger pre-release.
+set -eo pipefail
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+# shellcheck source=./../../lib/yaml.sh
+source "$DIR/../../lib/yaml.sh"
+# shellcheck source=./../../lib/bootstrap.sh
+source "$DIR/../../lib/bootstrap.sh"
+
+if [[ "$(yaml_get_field ".arguments.releaseOptions.enablePrereleases" "$(get_service_yaml)")" != "true" ]]; then
+  echo "releaseOptions.enablePrereleases is not true, skipping rc release"
+  exit 0
+fi
+
+# Default it to use main branch, should this be configurable?
+prereleaseBranch="main"
+# TODO: allow to config release user in service.yaml
+releaseUsername="Outreach CI"
+releaseUseremail="outreach-ci@outreach.io"
```

Comment:
issue: perhaps the default values for this should be in the box config. We should avoid Outreach-specific values in this repo since it's public.